src/collections/_documentation/performance/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Performance Monitoring
+sidebar_order: 0
+---
+
+Performance monitoring helps developers measure Apdex, Throughput, and trace slow transactions down to the poor performing API call or DB query. Available for both JavaScript and Python SDKs, performance monitoring helps you both diagnose problems and measure your application's overall health. 
+
+Learn more about performance monitoring in:
+
+- [Homepage]({% link _documentation/performance/performance-homepage.md %}): displays a high-level report of application health, including problematic transactions and both Apdex and Throughput graphs
+- [Distributed Tracing]({% link _documentation/performance/distributed-tracing.md %}): diagnose problems and view interactions across your software systems
+- [Metrics]({% link _documentation/performance/performance-metrics.md %}): provides insight into user experience of your application based on custom thresholds
+- [Discover]({% link _documentation/performance/discover/index.md %}): view data across environments with pre-built queries. Customers subscribed to the Team or Business plan can use Discover to view comprehensive information sent to Sentry.
+
+## Getting Started with Performance for JavaScript
+
+To get started with performance monitoring, first install the `@sentry/tracing package:`
+
+```bash
+npm install --save @sentry/tracing
+```
+
+Next, initialize the integration in your call to `Sentry.init`:
+
+```jsx
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/tracing';
+Sentry.init({
+  dsn: '<https://bc367fd982144d75923219dca8f35ad1@o1.ingest.sentry.io/2783252>',
```

Comment:
This looks like a hardcoded DSN and Project got in here? Let's figure out how this is templatized ... or even just put in example values until then.

Reply:
Fixed. Thanks.